packages/web-runtime/src/pages/account.vue

```diff
@@ -398,6 +406,10 @@ export default defineComponent({
       }
     }
 
+    const onUpdateTheme = () => {
+      showMessage({ title: $gettext('Preference saved.') })
+    }
```

Comment:
Why handle this here via events? I'd expect the `ThemeSwitcher` to handle the message 😅

Reply:
Why would you expect that?

Reply:
Because it seems unnecessarily complicated to me 🤷 It's not that we're updating data or something in the parent component which would justify that. I'd rather have all theme-logic decoupled in the ThemeSwitcher component (same like e.g. the GDPRExport component handles all gdpr related stuff).